quickwit/quickwit-proto/src/codegen/quickwit/quickwit.ingest.rs

```diff
@@ -53,17 +53,17 @@ impl CommitTypeV2 {
     /// (if the ProtoBuf definition does not change) and safe for programmatic use.
     pub fn as_str_name(&self) -> &'static str {
         match self {
-            CommitTypeV2::Auto => "Auto",
-            CommitTypeV2::Wait => "Wait",
-            CommitTypeV2::Force => "Force",
+            CommitTypeV2::Auto => "AUTO",
```

Comment:
Can we align the way we serialize/deserialize JSON and the string name we have here?
In json we used `snake_case` but this is `uppercase`.